woocommerce/woocommerce
# Woo AI - fix store branding settings page regression.

### Changes proposed in this Pull Request:

Re-adds the Woo AI Settings Group and settings that were previously removed because of a messy merge conflict resolution.
When the Edit Product page is loaded with the Woo AI plugin, two API calls are made to get the "tone of voice" and the "store description" values from the WC REST API. Currently, on `trunk`, you'll see 404 errors to these endpoints because they don't exist, thus, the branding functionality on the client side is broken.
![Screen Shot on 2023-08-16 at 22:41:53](https://github.com/woocommerce/woocommerce/assets/6723003/793aaefc-7465-4ced-ad8a-b72f321d4694)

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to WooCommerce->Settings->Advanced->Features.
2. Enable the AI Features, change the tone of voice and add a description.
3. Go to Products->Add New with the Dev Tools open, verify that the two network requests return the correct values
4. Generate a description and verify that it used the tone of voice; you should also see a mention of the store description (include the store name in the description).

### Changelog entry




<details>

#### Significance


#### Type


#### Message Woo AI - Fix store branding settings retrieval for use with description generation.

#### Comment 

</details>
